Drv/LinuxI2CDriver/Events.xml

```diff
@@ -0,0 +1,89 @@
+<events>
+    <!-- Open error -->
+    <event id="0" name="GP_OpenError" severity="WARNING_HI" format_string = "Error opening GPIO device %d: %d (%s)" >
```

Comment:
All you events are using the GPIO naming convention.  You should rename these to be more clear for I2C